_releases/2022-01-06-1.3.0-released.md

```diff
@@ -14,6 +14,30 @@ In this release we included [XXX PRs since the 1.2.2 release](https://github.com
 
 ## Interpreter
 
+The awaited Crystal interpreter has been [merged](https://github.com/crystal-lang/crystal/pull/11159). To use it, you need to compile Crystal with a special flag. The official releases (.deb, .rpm, docker images, etc.) are **not** being compiled with it, since its [missing substantial features](https://github.com/crystal-lang/crystal/issues/11555) for real use cases.
+
+The following is an extract of the [post](https://crystal-lang.org/2021/12/29/crystal-i.html) where we introduced this feature.
+Assuming Crystal was compiled with the flag `interpreter=1`, you can invoke the interpreter using two modes: `crystal i` or `crystal i file.cr`. In the first case, a repl will start right away, and in the second case the `file.cr` will be interpreted. Interestingly, if the interpreter finds the `debugger` instruction, it will start an interactive process to debug it. For instance, if we have a file `example.cr` with:
```

Comment:
```suggestion
You can invoke the interpreter using two modes: `crystal i` or `crystal i file.cr`. In the first case, a REPL will start right away, and in the second case the `file.cr` will be interpreted. Interestingly, if the interpreter finds the `debugger` instruction, it will start an interactive process to debug it. For instance, if we have a file `example.cr` with:
```